.ci/azure-pipelines/build_windows.bat

```diff
@@ -38,7 +38,7 @@
 if defined MUMBLE_BUILD_NUMBER_TOKEN (
 	:: The method we use to store a command's output into a variable:
 	:: https://stackoverflow.com/a/6362922
-	for /f "tokens=* USEBACKQ" %%g in (`python "scripts/mumble-version.py" --project`) do (set "VERSION=%%g")
+	for /f "tokens=* USEBACKQ" %%g in (`python "scripts\mumble-version.py" --project`) do (set "VERSION=%%g")
```

Comment:
Doesn't the backslash need escaping?

Reply:
Only in instances where a single backlash marks the beginning of an escape sequence (i.e. C).